### PR TITLE
add limitation

### PR DESCRIPTION
**Pull request by binoue**
#117 .

### CODE REVIEW

File: docs/limitations.md [hunk 1]
```diff
@@ -26,3 +26,9 @@ StorageClass mountOptions
 -------------------------
 
 TopoLVM does not recognize `mountOptions` of `StorageClass` currently.
+
+Time-lag in updating node capacity
```
Comment (ymmt2005) [inline docs/limitations.md]:
```suggestion
Capacity-aware scheduling may go wrong
```

File: docs/limitations.md [hunk 2]
```diff
@@ -26,3 +26,9 @@ StorageClass mountOptions
 -------------------------
 
 TopoLVM does not recognize `mountOptions` of `StorageClass` currently.
+
+Time-lag in updating node capacity 
+-------------------------
+
+Node capacity is not updated in TopoLVM's extended scheduler.
```
Comment (ymmt2005) [inline docs/limitations.md]: What capacity?  Storage?

File: docs/limitations.md [hunk 3]
```diff
@@ -26,3 +26,9 @@ StorageClass mountOptions
...
+Therefore, if many volumes are requested at the same time, they may be assigned to one node and cannot be created due to the shortage of node storage.
```
Comment (ymmt2005) [inline docs/limitations.md]:
```suggestion
Therefore, when multiple pods requesting TopoLVM volumes are requested at once, the extended scheduler cannot reference the exact capacity of the underlying LVM volume group.
```

File: docs/limitations.md [hunk 4]
```diff
@@ -26,3 +26,12 @@ StorageClass mountOptions
 -------------------------
 
 TopoLVM does not recognize `mountOptions` of `StorageClass` currently.
+
+Capacity-aware scheduling may go wrong
+-------------------------
+
+Node storage capacity annotation is not updated in TopoLVM's extended scheduler.
+Therefore, when multiple pods requesting TopoLVM volumes are requested at once, the extended scheduler cannot reference the exact capacity of the underlying LVM volume group.
```
Comment (ymmt2005) [inline docs/limitations.md]:
```suggestion
Therefore, when multiple pods requesting TopoLVM volumes are created at once, the extended scheduler cannot reference the exact capacity of the underlying LVM volume group.
```

**Comment by vovik0134** [comment]
I recently ran into this issue and it felt a little bit unpleasant. I wonder is there any possibility to avoid this issue or completeley solve

**Pull request merged**